src/components/addNewBook.js

```diff
@@ -1,58 +1,93 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv } from 'uuid';
-import { addBook, removeBook } from '../redux/books/books';
-import store from '../redux/configureStore';
-import BookForm from './inputBook';
-import Books from './Books';
+import { addBook } from '../redux/books/books';
+// import store from '../redux/configureStore';
+// import BookForm from './inputBook';
+// import Books from './Books';
+
+const AddBook = () => {
+  const categories = [
+    {
+      id: 1,
+      name: 'Action',
+    },
+    {
+      id: 2,
+      name: 'Science fiction',
+    },
+    {
+      id: 3,
+      name: 'Computer Science',
+    },
+  ];
 
-const BookList = () => {
   const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+
+  const lblBookTitleChanged = (e) => {
+    e.preventDefault();
+    setTitle(e.target.value);
+  };
 
-  const [books, setBooks] = useState(store.getState().bookReducer);
+  const chkCategoriesChanged = (e) => {
+    e.preventDefault();
+    setCategory(e.target.value);
+  };
 
-  const submitBook = ({ title, author }) => {
+  const Submit = () => {
     const newBook = {
-      id: uuidv(), // make sure it's unique
+      id: uuidv(),
       title,
-      author,
+      category,
     };
 
     dispatch(addBook(newBook));
```

Comment:
The user is not able to add the book to the list. Please make sure that books are being added and shown on the page using the API provided.